docs/security.md

```diff
@@ -96,13 +96,150 @@ The Dockerfile is configured to run as a non-root user:
 - Secure log storage and transmission
 - Implement log rotation and retention policies
 
+### Audit Logging
+
+The LLM Proxy provides comprehensive audit logging for security-sensitive operations, designed for compliance and security investigations.
+
+#### Configuration
+
+Audit logging is controlled by these environment variables:
+
+- **`AUDIT_ENABLED`** (default: `true`): Enable/disable audit logging entirely
+- **`AUDIT_LOG_FILE`** (default: `./data/audit.log`): Path to audit log file
+- **`AUDIT_CREATE_DIR`** (default: `true`): Create parent directories if they don't exist
+- **`AUDIT_STORE_IN_DB`** (default: `true`): Store audit events in database for analytics
+
+#### Enabling and Disabling
+
+```bash
+# Enable audit logging (default)
+AUDIT_ENABLED=true
+AUDIT_LOG_FILE=./data/audit.log
+AUDIT_STORE_IN_DB=true
+
+# Disable audit logging
+AUDIT_ENABLED=false
+
+# File-only audit logging (no database storage)
+AUDIT_ENABLED=true
+AUDIT_LOG_FILE=./data/audit.log
+AUDIT_STORE_IN_DB=false
+
+# Database-only audit logging (no file)
+AUDIT_ENABLED=true
+AUDIT_LOG_FILE=""
+AUDIT_STORE_IN_DB=true
+```
+
+#### Privacy and Data Protection
+
+**Token Obfuscation Guarantees:**
+- All tokens in audit logs are automatically obfuscated using a consistent pattern
+- Original tokens are never stored in audit logs in plaintext
+- Obfuscation preserves prefix (e.g., `tok-`) and shows partial content for identification
+- For tokens with known prefixes (such as `tok-`, `sk-`, `api_`, `Bearer `, or `ghp_`): the obfuscation logic shows the first 4 and last 4 characters of the token, with asterisks in the middle.
+- Example: `tok-1234567890abcdef` becomes `tok-1234****cdef`
```

Comment:
Expanded token obfuscation to define known prefixes (`tok-`, `sk-`, `api_`, `Bearer `, `ghp_`) and behavior for unknown prefixes with example.